Option --insecure not correctly handled

Since #25  (?) , when option `-- insecure` is set, newman fail with error message 
```
2018-04-03T01:09:56.6819866Z [command]C:\Windows\system32\cmd.exe /D /S /C "C:\npm\prefix\newman.cmd run --insecure -r "cli,html,json,junit" -n 1 --globals D:\a\1\s\PostmanTests\globals.postman_globals.json -e D:\a\1\s\PostmanTests\APITest.postman_environment.json D:\a\1\s\PostmanTests\APITest.postman_collection.json"
2018-04-03T01:09:58.1779293Z ENOENT: no such file or directory, open 'D:\a\1\s\--insecure'
2018-04-03T01:09:58.1779746Z unable to read data from file "--insecure"
2018-04-03T01:09:58.1779873Z { code: 1,
2018-04-03T01:09:58.1779988Z   error: undefined,
2018-04-03T01:09:58.1780142Z   stdout: 'unable to read data from file "--insecure"\n',
2018-04-03T01:09:58.1780303Z   stderr: 'ENOENT: no such file or directory, open \'D:\\a\\1\\s\\--insecure\'\n' }
2018-04-03T01:09:58.1780912Z ##[error]Failed
2018-04-03T01:09:58.1851312Z ##[section]Finishing: Shared
```
It seems that insecure option *can't* be set as first parameter, maybe an newman issue, but need to be fixed here as well.

--- a/NewmanPostman/newmantask.ts
+++ b/NewmanPostman/newmantask.ts
@@ -2,7 +2,7 @@ import path = require('path');
 import tl = require('vsts-task-lib/task');
 import trm = require('vsts-task-lib/toolrunner');
 
-function GetToolRunner(collectionToRun) {
+function GetToolRunner(collectionToRun:string) {
     var newman: trm.ToolRunner = tl.tool(tl.which('newman', true));
     newman.arg('run');
     newman.arg(collectionToRun);

--- a/NewmanPostman/newmantask.ts
+++ b/NewmanPostman/newmantask.ts
@@ -2,9 +2,10 @@ import path = require('path');
 import tl = require('vsts-task-lib/task');
 import trm = require('vsts-task-lib/toolrunner');
 
-function GetToolRunner() {
+function GetToolRunner(collectionToRun) {
     var newman: trm.ToolRunner = tl.tool(tl.which('newman', true));
     newman.arg('run');
+    newman.arg(collectionToRun);
 
     let sslClientCert = tl.getPathInput('sslClientCert', false, true);
     newman.argIf(typeof sslClientCert != 'undefined' && sslClientCert, ['--ssl-client-cert', sslClientCert]);
@@ -78,9 +79,7 @@ async function run() {
 
             if (matchedFiles.length > 0) {
                 matchedFiles.forEach((file: string) => {
-                    var newman: trm.ToolRunner = GetToolRunner();
-                    newman.arg(file);
-
+                    var newman: trm.ToolRunner = GetToolRunner(file);
                     var execResponse = newman.execSync();
 
                     if (execResponse.code === 1) {
@@ -95,9 +94,7 @@ async function run() {
             }
         }
         else {
-            var newman: trm.ToolRunner = GetToolRunner();
-            newman.arg(collectionFileSource);
-
+            var newman: trm.ToolRunner = GetToolRunner(collectionFileSource);
             await newman.exec();
         }
 

